### PR TITLE
Cloudinary setup

### PR DESCRIPTION
**Pull request by Martindevelops-songa**
Rider documents uploading and getting now working.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ app.use("/api/users/auth", UserAuthRoutes);
 app.use("/api/riders/auth", RiderAuthRoutes);
 app.use("/api/points", mpesaRoutes);
 app.use("/api/users/auth/OTP", UserOTPRoutes);
-app.use("/api/users/customer_agent", CustomerCareAgentRoutes);
+//app.use("/api/users/customer_agent", CustomerCareAgentRoutes);
```
Comment (dennisngeno-songa) [inline src/app.ts:25]: Kindly uncomment this one, we need the changes for customer care agents

File: src/controllers/RiderDocumentsControllers.ts [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    })) as RiderDocuments;
+    if (hasDocuments) {
+      //update the current docs
+      if (Array.isArray(images)) {
+        console.log("array");
+
+        for (const image of images) {
+          const parts = image.originalname.split(".");
+          const docName = parts[0];
+          console.log(docName);
```
Comment (dennisngeno-songa) [inline src/controllers/RiderDocumentsControllers.ts:59]: Would you remove console logs and use the logger instead

File: uploads/ID-back.png.png
Comment (dennisngeno-songa) [inline uploads/ID-back.png.png:1]: Are we supposed to add images to the source code or to Cloudinary?

**Review by dennisngeno-songa — CHANGES_REQUESTED** [review]

File: src/controllers/RiderDocumentsControllers.ts [hunk 3]
```diff
@@ -0,0 +1,182 @@
+import { PrismaClient, Rider, RiderDocuments } from "@prisma/client";
```
Comment (dennisngeno-songa) [inline src/controllers/RiderDocumentsControllers.ts:1]: Rider is not used

File: src/controllers/RiderDocumentsControllers.ts [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+//import { checkRider } from "../helpers/user";
```
Comment (dennisngeno-songa) [inline src/controllers/RiderDocumentsControllers.ts:2]: Would you remove the comment

File: src/controllers/RiderDocumentsControllers.ts [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+          const docName = parts[0];
+          console.log(docName);
+          //result object contains {docName, secureUrl}- this will be stored in the db
+          const result = (await UploadToCloudinary({
+            username: `${rider?.first_name} ${rider?.last_name}`,
+            image: image.path,
+            userId: rider!.id,
+            docName: docName,
+          })) as UploadResult;
+          console.log(result, "result");
```
Comment (dennisngeno-songa) [inline src/controllers/RiderDocumentsControllers.ts:67]: Code has been duplication. Would you make it DRY

Comment (dennisngeno-songa) [reply src/controllers/RiderDocumentsControllers.ts:67]: The function is also long

**Review by Martindevelops-songa — COMMENTED** [review]
good

**Review by mog-rn — APPROVED** [review]

File: src/controllers/RiderDocumentsControllers.ts [hunk 6]
```diff
@@ -0,0 +1,182 @@
+import { PrismaClient, Rider, RiderDocuments } from "@prisma/client";
+import { checkRider } from "../helpers/user";
```
Comment (mog-rn) [inline src/controllers/RiderDocumentsControllers.ts:2]: There is aduplicate identifier here an also on ine 4 for the checkRider.

**Comment by dennisngeno-songa** [comment]
Still seeing images and console logs in the diff

**Comment by dennisngeno-songa** [comment]
and faling tests

**Pull request closed without merging**